xml/System.Reflection/PropertyInfo.xml

```diff
@@ -104,7 +104,7 @@
 >  This example generates about 55,000 lines of data, which you can redirect to a text file at the command prompt, as follows: **example.exe > propertyinfo.txt**  
   
  :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_CLR/Reflection/CPP/reflection.cpp" id="Snippet1":::
- :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR/Reflection/CS/Reflection.cs" id="Snippet1":::
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR/Reflection/CS/Reflection.cs" interactive="try-dotnet" id="Snippet1":::
```

Comment:
> Add Run button for PropertyInfo.PropertyType Examples

This is not `PropertyInfo.PropertyType` Example. 

Also see note above `This example generates about 55,000 lines of data` i don't think it is appropriate for running in the browser